workflow-manager/main.go

```diff
@@ -111,16 +110,78 @@ func work(inputBucket, k8sNS string) error {
 		names = append(names, attrs.Name)
 	}
 
-	var readybatches []*batch
-	var unreadybatches []*batch
+	var readyBatches []string
 	for k, v := range batches {
 		if v.metadata && v.avro && v.sig {
 			log.Printf("ready: %s", k)
-			readybatches = append(readybatches, v)
+			readyBatches = append(readyBatches, k)
 		} else {
 			log.Printf("unready: %s", k)
-			unreadybatches = append(unreadybatches, v)
 		}
 	}
+
+	log.Printf("starting %d jobs", len(readyBatches))
+	for _, batchName := range readyBatches {
+		err = startJob(ctx, clientset, batchName)
+		if err != nil {
+			return fmt.Errorf("starting job for batch %q: %w", batchName, err)
+		}
+	}
+	return nil
+}
+
+func startJob(ctx context.Context, clientset *kubernetes.Clientset, batchName string) error {
+	log.Printf("starting job for batch %q...", batchName)
+	jobName := fmt.Sprintf("i-batch-%s", strings.ReplaceAll(batchName, "/", "-"))
+
+	pathComponents := strings.Split(batchName, "/")
+	batchID := pathComponents[len(pathComponents)-1]
+	batchDate := strings.Join(pathComponents[0:len(pathComponents)-1], "/")
+	job := &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      jobName,
+			Namespace: *k8sNS,
+		},
+		Spec: batchv1.JobSpec{
+			Template: corev1.PodTemplateSpec{
+				Spec: corev1.PodSpec{
+					RestartPolicy: "Never",
+					Containers: []corev1.Container{
+						{
+							Args: []string{
+								"batch-intake",
+								"--s3-use-credentials-from-gke-metadata",
+								"--aggregation-id", "fake-1",
+								"--batch-id", batchID,
+								"--date", batchDate,
+								"--ingestion-bucket", "gs://tbd",
+								"--validation-bucket", "gs://tbd",
+								"--ingestor-public-key", "what_is_my_pubkey",
```

Comment:
The value for `ingestion-bucket` is the `input-bucket` value this program got.

`validation-bucket` must be retrieved from the peer data share processor's relevant specific manifest. To find it, facilitator will need to be provided their manifest hosting base URL (i.e. wire `var.peer_share_processor_manifest_domain` from the top-level Terraform module into facilitator) and the data share processor name (i.e., wire `var.data_share_processor_name` from data_share_processor.tf into facilitator).

`ingestor-public-key` must be retrieved from the ingestor's global manifest. To find it, facilitator will need to be provided the relevant ingestor's manifest hosting base URL (so wire the appropriate value from the `var.ingestors` map in the top level TF module into the facilitator).